### PR TITLE
Feature/readme update

### PR DESCRIPTION
**Pull request by GaetanGrd**
## Scope

[closes TICKET-45](https://github.com/Zagrios/bs-manager/issues/45)

## Implementation

- Recreating a readme for the 1.0 release

## Screenshots

| before | after |
| ------ | ----- |
| image| image|

## How to Test

- [x] Review (by another) : 
> - [x] Check check if all the links have been set and working 
> - [x] Check if there are any missing sections
> - [x] Check Orthography
> - [x] Check Grammar

## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!

> 🟡 Why was the default value removed?

E.g: Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | RED |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,231 @@
...
+    <span>
+        The "One-Click" feature of <a href="../../">BSManager</a> is an incredibly convenient tool for Beat Saber users. It allows for easy and quick installation of models for all existing versions of Beat Saber. With this system, you will no longer need to spend hours manually installing models for each specific version of Beat Saber. All you need is one simple click to install models on all versions of Beat Saber you own.
+    </span>
+</div>
+<br>
+<div align="center">
+
+
+
+https://user-images.githubusercontent.com/40648115/213423435-f0fc9dda-1328-449f-ab54-39d28cb6a8ac.mov
```
Comment (Zagrios) [inline README.md:216]: 🔴Where is the oneClick downloads for Maps and playlist ?

**Review by Zagrios — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Zagrios) [reply README.md:216]: you can click on the images listed in the oneClicks cards (where you enable, disable the oneClick)

Comment (GaetanGrd) [reply README.md:216]: so amazing, thanks

Comment (GaetanGrd) [reply README.md]: fix

File: README.md [hunk 2]
```diff
@@ -1,76 +1,229 @@
...
+    <i>(Beat Saber Manager)</i>
+    <br><br>
+    <span>
+        Download, manage and customize your versions of Beat Saber with a simple click!
+    </span>
+    <br>
+    <span>
+        <a href="https://github.com/Zagrios/bs-manager/issues/new?assignees=Zagrios&labels=bug&template=bug_report.md&title=%5BBUG%5D">Report Bug</a> 
+        ·
+        <a href="https://github.com/Zagrios/bs-manager/issues/new?assignees=Zagrios&labels=enhancement&template=feature_request.md&title=%5BAME%5D">Request Feature</a>
```
Comment (Zagrios) [inline README.md:29]: 🔴wrong link

File: README.md [hunk 3]
```diff
@@ -1,76 +1,229 @@
...
+    <a href="https://github.com/Zagrios/bs-manager/blob/master/LICENSE"><img src ="https://img.shields.io/github/license/Zagrios/bs-manager?style=for-the-badge" alt="License"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/graphs/contributors"><img src ="https://img.shields.io/github/contributors/Zagrios/bs-manager?style=for-the-badge" alt="Contributors"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/issues"><img alt="GitHub issues" src="https://img.shields.io/github/issues/Zagrios/bs-manager?style=for-the-badge"></a>
+    <br>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-DISCORD-5865f2?style=for-the-badge&logo=discord&logoColor=ffffff" alt="discord"/></a>
+    <a href="https://mee6.xyz/fr/m/bsmanager"><img src ="https://img.shields.io/badge/-🥰%20Support%20BSM-EC4546?style=for-the-badge" alt="Donation"/></a>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-Twitter-F5F8FA?style=for-the-badge&logo=Twitter" alt="Twitter"/></a>
+    <br>
+    [<a href="#readme-bot">go to bot</a>]
```
Comment (Zagrios) [inline README.md:47]: 🟡 bot ? not bottom ?

File: README.md (hunk 2 above)
Comment (GaetanGrd) [reply README.md:29]:
```suggestion
        <a href="https://github.com/Zagrios/bs-manager/issues/new?assignees=Zagrios&labels=enhancement&template=-feat---feature-request.md&title=%5BFEAT.%5D+%3A+">Request Feature</a>
        ·
        <a href="https://github.com/Zagrios/bs-manager/security/policy">Report a security vulnerability</a>
```

Comment (GaetanGrd) [reply README.md]: f7da6fdb6389ba3f2d7aa9adef1b7b9da9dc3709

File: README.md (hunk 3 above)
Comment (GaetanGrd) [reply README.md:49]: By saying "bot" all the time I forgot that the full word was "bottom" XD

```suggestion
    [<a href="#readme-bot">go to bottom</a>]
```

File: README.md [hunk 4]
```diff
@@ -1,76 +1,229 @@
...
+        <ul>
+            <li><a href="#built-with">Built With</a></li>
+            <li><a href="#contributing">Contributing</a></li>
+        </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <ul>
+        <li><a href="#how-to-install">How to install?</a></li>
+        <li><a href="#manage-versions">Manage Versions</a></li>
+        <li><a href="#customize-your-instances">Customize your instances</a></li>
```
Comment (Zagrios) [inline README.md:67]: Customize your versions

Comment (GaetanGrd) [reply README.md:67]:
```suggestion
        <li><a href="#customize-your-instances">Customize your versions</a></li>
```

File: README.md [hunk 5]
```diff
@@ -1,76 +1,229 @@
...
-	<img src="https://github.com/Zagrios/bs-manager/blob/master/resources/readme/beat-conflict.png?raw=true" width="130"/>
-</p>
 
+<!-- ABOUT THE PROJECT -->
+<div align="center">
+    <h1> </h1>
+    <h1><b>About The Project</b></h1>
+</div>
+<div>
+    <a href="../../">BSManager</a> was created to unify and improve the two most used tools in the community (<a href="https://github.com/RiskiVR/BSLegacyLauncher">BS legacy launcher</a> and <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>).
```
Comment (Zagrios) [inline README.md:81]: 🔴All BSManager links not work

File: README.md [hunk 6]
```diff
@@ -1,76 +1,231 @@
...
+        <li><b>Download your maps in the same way</b>, in addition to that, the maps are not version dependent, so you will be able to find your maps whatever version you are using without having to transfer them.</li>
+        <li><b>Install your mods in one click</b>, just like <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>, <a href="https://github.com/Zagrios/bs-manager">BSManager</a> will allow you to install your mods quickly and securely depending on the version you choose.</li>
+        <li><b>Customize your interface</b>, we know that some of you like to customize your interfaces to the maximum. That's why <a href="https://github.com/Zagrios/bs-manager">BSManager</a> brings you peace of mind on this point, you can change the name of the versions, their color, and you can even change the color of the logo! Of course, a dark theme has been added to protect your eyes at night.</li>
+    </ul>
+</div>
+<br>
+<div align="center">
+    <img src="resources/readme/preview.png" alt="(WIP) home interface with multiple instance dl and steam & oculus.">
```
Comment (Zagrios) [inline README.md:93]: 🟡 WIP ?

File: README.md (hunk 5 above)
Comment (GaetanGrd) [reply README.md]: f6159a863c5d59e720960f440bca8f6647ef5196

File: README.md (hunk 6 above)
Comment (GaetanGrd) [reply README.md]: b23e211e110ba68a5316a172d4cc5ea96781bc7c

File: README.md [hunk 7]
```diff
@@ -1,76 +1,231 @@
...
+    <h1> </h1>
+    <h1><b>About The Project</b></h1>
+</div>
+<div>
+    <a href="https://github.com/Zagrios/bs-manager">BSManager</a> was created to unify and improve the two most used tools in the community (<a href="https://github.com/RiskiVR/BSLegacyLauncher">BS legacy launcher</a> and <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>).
+    <br><br>
+    <h4><a href="https://github.com/Zagrios/bs-manager">BSManager</a> will allow you to:</h4>
+    <ul>
+        <li><b>Manage instances of multiple versions</b>, designed for people who need to play on different versions or have multiple profiles for the same version.</li>
+        <li><b>Download your maps in the same way</b>, in addition to that, the maps are not version dependent, so you will be able to find your maps whatever version you are using without having to transfer them.</li>
```
Comment (Zagrios) [inline README.md:86]: 🔴  depending on your choice maps are not version dependent

File: README.md [hunk 8]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+    <h1> </h1>
+    <h1><b>About The Project</b></h1>
+</div>
+<div>
+    <a href="https://github.com/Zagrios/bs-manager">BSManager</a> was created to unify and improve the two most used tools in the community (<a href="https://github.com/RiskiVR/BSLegacyLauncher">BS legacy launcher</a> and <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>).
+    <br><br>
+    <h4><a href="https://github.com/Zagrios/bs-manager">BSManager</a> will allow you to:</h4>
+    <ul>
+        <li><b>Manage instances of multiple versions</b>, designed for people who need to play on different versions or have multiple profiles for the same version.</li>
```
[the remark this answers is not in the document]
Comment (GaetanGrd) [reply README.md:85]:
```suggestion
        <li><b>Manage multiple BeatSaber versions</b>, designed for people who need to play on different versions or have multiple profiles for the same version.</li>
```

File: README.md [hunk 9]
```diff
@@ -1,76 +1,231 @@
...
+        <li>
+            <a href="https://tailwindcss.com/"><img src="https://img.shields.io/badge/-Tailwind%20CSS-white?style=for-the-badge&logo=Tailwind%20CSS" alt="TailWind CSS"></a>
+        </li>
+    </ul>
+</div>
+<br>
+
+<!--CONTRIBUTING-->
+<div>
+    <h2><b>Contributing</b></h2>
```
Comment (Zagrios) [inline README.md:119]: Contribution, Makers, Owners, i dont know but contributing is stange

File: README.md [hunk 10]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+    <h1> </h1>
+    <h1><b>Usage</b></h1>
+</div>
+
+<!--HOW TO INSTALL?-->
+<div>
+    <h2><b>How to install?</b></h2>
+    <ul>
+        <li>Download the latest release from <a href="https://github.com/Zagrios/bs-manager/releases">Releases</a>.</li>
```
Comment (Zagrios) [inline README.md:147]: 🟡 maybe redirect to https://github.com/Zagrios/bs-manager/releases/latest

File: README.md (hunk 7 above)
Comment (GaetanGrd) [reply README.md:86]:
```suggestion
        <li><b>Download your maps in the same way</b>, in addition to that, depending on your choice, maps are not version dependent. so you will be able to find your maps whatever version you are using without having to transfer them.</li>
```

File: README.md (hunk 10 above)
Comment (Zagrios) [reply README.md:147]: no tag will change

Comment (GaetanGrd) [reply README.md:147]:
```suggestion
        <li>Download the <a href="https://github.com/Zagrios/bs-manager/releases/latest">latest release</a> from <a href="https://github.com/Zagrios/bs-manager/releases">Releases</a>.</li>
```

that* 

my bad, I didn't notice the link

Comment (Zagrios) [reply README.md:147]: yes

File: README.md (hunk 9 above)
Comment (GaetanGrd) [reply README.md:119]:
```suggestion
    <h2><b>Contribution</b></h2>
```

File: README.md [hunk 11]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213008824-25cfdda3-6350-42dc-aa53-2132b4b8e838.mov
+</div>
+
+<!--CUSTOM UR INSTANCES-->
+<div>
+    <h2><b>Customize your instances</b></h2>
+    <span>
+        The tool will allow you to completely customize each of your downloadable versions, you can change the color of the icon, choose a name for the instance and finally install your mods and map according to the Beat Saber version!
```
Comment (Zagrios) [inline README.md:171]: 🔴 each of your "downloaded" versions
🟡 To tool will ... -> BSManager ... (no will here)
🟡 Choose a name for the "version" (instance is a technical term)
🟡 mods, maps, and other contents  ....

File: README.md [hunk 12]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296816-bdaafc90-da76-4835-8d83-9ac6a4e1c549.mov
+</div>
+
+<!--INSTALL MAPS-->
+<div>
+    <h2><b>Install Maps</b></h2>
+    <span>
+        In addition to managing mods, the tool that allows you to manage and organize custom beatmaps existing on <a href="https://beatsaver.com">Beat Saver</a>, a popular site for sharing custom beatmaps for beat Saber. The tool allows you to view and organize your beatmaps, as well as link different versions of the game to share all your maps together.
```
Comment (Zagrios) [inline README.md:197]: 🟡 the tool -> the app

File: README.md (hunk 11 above)
Comment (GaetanGrd) [reply README.md:171]: fixed

File: README.md (hunk 12 above)
Comment (GaetanGrd) [reply README.md]: all "tool" removed xD

File: README.md [hunk 13]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296816-bdaafc90-da76-4835-8d83-9ac6a4e1c549.mov
+</div>
+
+<!--INSTALL MAPS-->
+<div>
+    <h2><b>Install Maps</b></h2>
+    <span>
+        In addition to managing mods, <a href="https://github.com/Zagrios/bs-manager">BSManager</a> that allows you to manage and organize custom beatmaps existing on <a href="https://beatsaver.com">Beat Saver</a>, a popular site for sharing custom beatmaps for beat Saber. <a href="https://github.com/Zagrios/bs-manager">BSManager</a> allows you to view and organize your beatmaps, as well as link different versions of the game to share all your maps together.
```
Comment (Zagrios) [inline README.md:197]: 🔴  No "that" in "that allows you to manage"
🔴 Second sentence is useless replace with : `Moreover the app allow you to share your maps across all your versions`

File: README.md [hunk 14]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296838-f775fe83-866d-4764-a7d0-648dbb8ec983.mov
+</div>
+
+<!--ONE CLICK-->
+<div>
+    <h2><b>One Click</b></h2>
+    <span>
+        The "One-Click" feature of <a href="https://github.com/Zagrios/bs-manager">BSManager</a> is an incredibly convenient app for Beat Saber users. It allows for easy and quick installation of models for all existing versions of Beat Saber. With this system, you will no longer need to spend hours manually installing models for each specific version of Beat Saber. All you need is one simple click to install models on all versions of Beat Saber you own.
```
Comment (Zagrios) [inline README.md:210]: 🟡 OneClick is not only for models 🤔 
🟡 tool -> feature

Comment (GaetanGrd) [reply README.md:210]:
```suggestion
        The "One-Click" feature of <a href="https://github.com/Zagrios/bs-manager">BSManager</a> is an incredibly convenient feature for Beat Saber users. It allows for easy and quick installation of models,maps and playlist for all existing versions of Beat Saber. With this system, you will no longer need to spend hours manually installing models for each specific version of Beat Saber. All you need is one simple click to install models on all versions of Beat Saber you own.
```

File: README.md (hunk 13 above)
Comment (GaetanGrd) [reply README.md:197]:
```suggestion
        In addition to managing mods, <a href="https://github.com/Zagrios/bs-manager">BSManager</a> allows you to manage and organize custom beatmaps existing on <a href="https://beatsaver.com">Beat Saver</a>, a popular site for sharing custom beatmaps for beat Saber. Moreover the app allow you to share your maps across all your versions. 
```

?

File: README.md [hunk 15]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296838-f775fe83-866d-4764-a7d0-648dbb8ec983.mov
+</div>
+
+<!--ONE CLICK-->
+<div>
+    <h2><b>One Click</b></h2>
+    <span>
+        The "One-Click" feature of <a href="https://github.com/Zagrios/bs-manager">BSManager</a> is an incredibly convenient feature for Beat Saber users. It allows for easy and quick installation of models,maps and playlist for all existing versions of Beat Saber. With this system, you will no longer need to spend hours manually installing models for each specific version of Beat Saber. All you need is one simple click to install models on all versions of Beat Saber you own.
```
Comment (Zagrios) [inline README.md:210]:
```suggestion
        The "One-Click" feature of <a href="https://github.com/Zagrios/bs-manager">BSManager</a> is an incredibly convenient feature for Beat Saber users. It allows easy and quick installation of models, maps and playlists for all versions of Beat Saber you own.
```

File: README.md [hunk 16]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213435051-7feb815d-f32b-4135-b095-ef49726a836c.mov
+</div>
+
+<!--HAVING PROBLEMS?-->
+<div>
+    <h2><b>Having problems?</b></h2>
+    <span>
+        See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> for a list of known issues, from there you can create a new issue for the problem you have!
```
Comment (Zagrios) [inline README.md:223]:
```suggestion
        See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> for a list of known issues, from there you can create a new issue for your problem!  
```

Comment (GaetanGrd) [reply README.md:223]: gougou gaga XD

File: README.md [hunk 17]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213435051-7feb815d-f32b-4135-b095-ef49726a836c.mov
+</div>
+
+<!--HAVING PROBLEMS?-->
+<div>
+    <h2><b>Having problems?</b></h2>
+    <span>
+        See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> for a list of known issues, from there you can create a new issue for your problem!
```
Comment (Zagrios) [inline README.md:223]:
```suggestion
        See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> show the list of known issues, from there you can create a new issue for your problem!  
```

Comment (Zagrios) [reply README.md:223]: to show

File: README.md [hunk 18]
```diff
@@ -1,76 +1,231 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213435051-7feb815d-f32b-4135-b095-ef49726a836c.mov
+</div>
+
+<!--HAVING PROBLEMS?-->
+<div>
+    <h2><b>Having problems?</b></h2>
+    <span>
+        See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> show the list of known issues, from there you can create a new issue for your problem!
```
Comment (Zagrios) [inline README.md:223]:
```suggestion
        See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> to show the list of known issues, from there you can create a new issue for your problem!  
```

File: README.md [hunk 19]
```diff
@@ -1,76 +1,231 @@
...
+
+https://user-images.githubusercontent.com/40648115/213435051-7feb815d-f32b-4135-b095-ef49726a836c.mov
+</div>
+
+<!--HAVING PROBLEMS?-->
+<div>
+    <h2><b>Having problems?</b></h2>
+    <span>
+        See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> show the list of known issues, from there you can create a new issue for your problem!  
+        Otherwise you can add on to existing issues if one matches what you're encountering.
```
Comment (Zagrios) [inline README.md:224]: useless sentence

Comment (GaetanGrd) [reply README.md:224]:
```suggestion

```

File: README.md [hunk 20]
```diff
@@ -1,76 +1,230 @@
...
+    <a href="https://github.com/Zagrios/bs-manager/blob/master/LICENSE"><img src ="https://img.shields.io/github/license/Zagrios/bs-manager?style=for-the-badge" alt="License"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/graphs/contributors"><img src ="https://img.shields.io/github/contributors/Zagrios/bs-manager?style=for-the-badge" alt="Contributors"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/issues"><img alt="GitHub issues" src="https://img.shields.io/github/issues/Zagrios/bs-manager?style=for-the-badge"></a>
+    <br>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-DISCORD-5865f2?style=for-the-badge&logo=discord&logoColor=ffffff" alt="discord"/></a>
+    <a href="https://mee6.xyz/fr/m/bsmanager"><img src ="https://img.shields.io/badge/-🥰%20Support%20BSM-EC4546?style=for-the-badge" alt="Donation"/></a>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-Twitter-F5F8FA?style=for-the-badge&logo=Twitter" alt="Twitter"/></a>
```
Comment (Zagrios) [inline README.md:47]: 🔴 it redirect to discord, not twitter x)

Comment (GaetanGrd) [reply README.md:47]:
```suggestion
    <a href="https://twitter.com/BSManager_"><img src ="https://img.shields.io/badge/-Twitter-F5F8FA?style=for-the-badge&logo=Twitter" alt="Twitter"/></a>
```

File: README.md [hunk 21]
```diff
@@ -1,76 +1,230 @@
...
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296794-74a1e20b-12dc-44ec-a67f-e25d7010bd3d.mov
+</div>
+
+<!--INSTALL MODS-->
+<div>
+    <h2><b>Install Mods</b></h2>
+    <span>
+        <a href="https://github.com/Zagrios/bs-manager">BSManager</a> allows you to manage Beat Saber mods in a quick and easy way. It uses the BeatMod API to allow you to install, update or uninstall mods for each version of your collection. So the app can help you manage them efficiently.
```
Comment (Zagrios) [inline README.md:184]: make `BeatMod` link to BeatMod

Comment (GaetanGrd) [reply README.md:184]:
```suggestion
        <a href="https://github.com/Zagrios/bs-manager">BSManager</a> allows you to manage Beat Saber mods in a quick and easy way. It uses the <a href="https://beatmods.com/#/">BeatMod API</a> to allow you to install, update or uninstall mods for each version of your collection. So the app can help you manage them efficiently.
```

**Review by Zagrios — APPROVED** [review]
👏 👏

**Pull request merged**